src/main/java/com/easypost/model/Event.java

```diff
@@ -3,14 +3,17 @@
 import com.easypost.exception.EasyPostException;
 import com.easypost.net.EasyPostResource;
 
+import java.util.List;
 import java.util.Map;
 
 public final class Event extends EasyPostResource {
     private String id;
```

Comment:
Question: Have we not moved `id` and `mode` to the base `EasyPostResource` class yet? If so, they're not needed to be duplicated here.

Reply:
We only have the getters for `id` and `mode` in `EasyPostResource`, and they are returning an empty string which is incorrect. We have to address this in a different PR.